include/io/controller.h

```diff
@@ -3,6 +3,36 @@
 
 #include <PR/ultratypes.h>
 
+#define CONT_A      0x8000
+#define CONT_B      0x4000
+#define CONT_G	    0x2000
+#define CONT_START  0x1000
```

Comment:
we recently reworked these defines (and all libultra related headers) in oot, not sure if the updated button defines want to be used here as well 

code looks good to me